superset-frontend/src/messageToasts/actions/index.ts

```diff
@@ -19,6 +19,8 @@
 import shortid from 'shortid';
 import { ToastType, ToastMeta } from '../types';
 
+type ToastOptions = Partial<Omit<ToastMeta, 'id' | 'toastType' | 'text'>>;
```

Comment:
Thanks for the refactor. but why `messageToasts` component is related with this PR?

Reply:
A new `noDuplicate` option was added in case the same verification endpoint returns the same error message. We could extract this change to a new PR but that PR would miss some context. Plus I'm also lazy.